pkg/kube/kube_handler.go

```diff
@@ -356,6 +363,16 @@ func NewHandler(kubeappsNamespace string, clustersConfig ClustersConfig) (AuthHa
 	if err != nil {
 		return nil, err
 	}
+	// Modify the default number of requests that the given client can do
+	// This is useful to handle a large number of namespaces which are check in parallel
+	// Burst is the initial number of request made in parallel
+	// Then further requests are performed following the QPS rate
+	if burst != 0 {
```

Comment:
Not sure why we're handling the case where burst or qps == 0? That would make sense if our chart values defaulted to zero and we were using the k8s defaults, but we're not, so this would only have an effect if the chart was explicitly configured with these values as 0, which would mean we'd not set them in the config and hence use the k8s defaults? Is that what you meant? Otherwise, I'd just set them unconditionally? But fine either way.

Reply:
yep, not needed, a leftover when it could be a env var.